pytradfri/__main__.py

```diff
@@ -122,12 +130,17 @@ def dump_devices() -> None:
     print("> devices")
     print("> homekit_id")
     if light:
+        print("> lights")
         print("> light.light_control.lights")
         print("> api(light.light_control.set_dimmer(10))")
         print("> api(light.light_control.set_dimmer(254))")
         print("> api(light.light_control.set_xy_color(30015, 26870))")
         print("> api(light.light_control.set_predefined_color('Warm Amber'))")
         print("> api(lights[1].light_control.set_dimmer(20))")
+    if socket:
+        print("> sockets")
+        print("> socket.socket_control.sockets")
+        print("> api(socket.socket_control.set_values({ const.ATTR_REPEAT_DAYS: 0b1111111 }))")
```

Comment:
We don't use the const module above.

Reply:
Do we need to change `__main__.py` to fix this bug? I don't mind appending this file, as it makes the library easier to use, but perhaps in another PR?

Reply:
I can do it in another PR, yes. I have been iterating just trying to get the tests to pass, I didn't realize there was already activity in this PR. I've stopped updating so far, but happy to move it out.

Reply:
OK. I split out #615 from this. I will force push to reduce down to just the bugfix.